src/test/java/org/broadinstitute/hellbender/utils/hmm/HeavyStateTestHMModel.java

```diff
@@ -18,13 +21,9 @@
     private final double lightStateProbs;
 
     public HeavyStateTestHMModel(final int numStates, final int heavyState, final double heavyStateWeight) {
-        if (numStates <= 1) {
-            throw new IllegalArgumentException("bad number of states");
-        } else if (heavyState < 0 || heavyState >= numStates) {
-            throw new IllegalArgumentException("bad heavy state");
-        } else if (heavyStateWeight <= (1.0 / numStates) || heavyStateWeight >= 1.0) {
-            throw new IllegalArgumentException("bad heavy state weight");
-        }
+        Utils.validateArg(numStates > 1, "bad number of states");
+        Utils.validIndex(heavyState, numStates);
+        ParamUtils.inRange(heavyStateWeight, (1.0 / numStates), 1.0, "bad heavy state weight");
```

Comment:
It's fine not to `throw` when ` heavyStateWeight == 1.0`.  The idea here is to have an HMM biased toward transitions to a particular "heavy" state.  /since the expected behavior is obvious, it's useful for testing.  If the weight is 1.0, it's complete bias -- the HMM will never go to another state, which is a valid extreme case.